### PR TITLE
Moderation settings

### PR DESCRIPTION
**Pull request by robkam**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3598,6 +3598,20 @@
 		'default' => true,
 		'trexwiki' => false,
 	),
+	
+	// Moderation settings
+	'wgModerationNotificationEnable' => array( 
+		'default' => false, // Disable notifications
+		'scruffywiki' => true,
```
Comment (Reception123) [inline LocalSettings.php]: You removed scruffywiki in your other PR

File: LocalSettings.php [hunk 2]
```diff
@@ -3598,6 +3598,20 @@
...
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array(
+		'default' => false, // Notify administrator only about new pages requests
+	),
+	'wgModerationEmail' => array(
+		'default' =>  $wgEmergencyContact, // Email address to send notifications
+		'sdiywiki' => admin@sdiy.info,
```
Comment (Reception123) [inline LocalSettings.php]: there needs to be a '' here too

File: LocalSettings.php [hunk 3]
```diff
@@ -3598,6 +3598,20 @@
...
+	'wgModerationNotificationEnable' => array( 
+		'default' => false, // Disable notifications
+		'scruffywiki' => true,
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array(
+		'default' => false, // Notify administrator only about new pages requests
+	),
+	'wgModerationEmail' => array(
+		'default' =>  $wgEmergencyContact, // Email address to send notifications
```
Comment (Reception123) [inline LocalSettings.php]: please remove the space between => and $wg

File: LocalSettings.php [hunk 4]
```diff
@@ -3598,6 +3598,20 @@
...
+		'default' => false, // Disable notifications
+		'scruffywiki' => true,
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array(
+		'default' => false, // Notify administrator only about new pages requests
+	),
+	'wgModerationEmail' => array(
+		'default' => $wgEmergencyContact, // Email address to send notifications
+		'sdiywiki' => admin@sdiy.info,
```
Comment (The-Voidwalker) [inline LocalSettings.php]: I believe this needs to be a string, eg `'admin@sdiy.info'`

Comment (robkam) [reply LocalSettings.php]: thanks

**Comment by JohnFLewis** [comment]
For the email, as we don't control it I'd be cautious. If there is any SPF or DMARC set up for the email domain I'd strongly recommended against this.

**Comment by robkam** [comment]
At the moment there are no notifications for new moderation tasks requiring attention. A banner message on whichever page is open would be more immediate and useful. I think enabling email is turn notifications on.

**Comment by JohnFLewis** [comment]
I meant against using emails not in our control, the default setting is acceptable though.

**Pull request closed without merging**